releasenotes/notes/add-device-profile-section-to-status-852cbd48d52d8c2d.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add a section in the agent status to show the device profile parsing errors.
```

Comment:
```suggestion
  - |
    Add a section in the Agent status to show device profile parsing errors.
```

Reply:
changed in this commit https://github.com/DataDog/datadog-agent/pull/25233/commits/97230d1ccffb1fff7226f1c4f502e6197165147b